Casks/f/frappe-books.rb

```diff
@@ -1,27 +1,37 @@
 cask "frappe-books" do
-  version "0.21.2"
-  sha256 "9fd0a360f35d9c0745ca43b459d133b7da555122005499a8372eb6fa90719723"
+  arch arm: "-arm64", intel: ""
 
-  url "https://github.com/frappe/books/releases/download/v#{version}/Frappe-Books-#{version}.dmg",
+  on_arm do
+    version "0.22.0"
+    sha256 "06ca67133f11f20b7aaff28f968d028db53b3e8483967493ff09a464c53fbbc6"
+
+    livecheck do
+      url :url
+      strategy :github_latest
+    end
+
+    depends_on macos: ">= :high_sierra"
+  end
+  on_intel do
+    version "0.21.2"
+    sha256 "9fd0a360f35d9c0745ca43b459d133b7da555122005499a8372eb6fa90719723"
+
+    livecheck do
+      skip "Legacy version for intel"
```

Comment:
```suggestion
      skip "Legacy version"
```

We typically do not make a distinction here.